src/main/java/com/strangeone101/holoitemsapi/enchantment/EnchantListener.java

```diff
@@ -100,120 +104,144 @@ public void onProjectileLaunch(ProjectileLaunchEvent event) {
         });
     }
 
+    /**
+     * Makes sure that after a player closes an anvil inventory, their instant build abilitiy gets disabled
+     */
+    @EventHandler(priority = EventPriority.MONITOR)
+    public void onAnvilClose(InventoryCloseEvent event) {
```

Comment:
For consistency, I would still like the method name to match the Bukkit event's name.